Services/DataX.Config/DataX.Config.Local.Test/Resource/flowSaved.json

```diff
@@ -69,150 +69,6 @@
           },
           "dynamicOffsetInMs": 300000
         }
-      },
```

Comment:
Did you fix fix the local E2E test as well?

Reply:
the local E2E test was originally disabled so dint touch that